itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -589,6 +590,72 @@ public void testIncrementalAdds() throws IOException {
   }
 
   @Test
+  public void testIncrementalLoadWithVariableLengthEventId() throws IOException, TException {
+    String testName = "incrementalLoadWithVariableLengthEventId";
+    String dbName = createDB(testName);
+    run("CREATE TABLE " + dbName + ".unptned(a string) STORED AS TEXTFILE");
+
+    String[] empty = new String[] {};
+    advanceDumpDir();
+    run("REPL DUMP " + dbName);
+    String replDumpLocn = getResult(0, 0);
+    String replDumpId = getResult(0, 1, true);
+    LOG.info("Bootstrap-Dump: Dumped to {} with id {}", replDumpLocn, replDumpId);
+    run("REPL LOAD " + dbName + "_dupe FROM '" + replDumpLocn + "'");
+
+    // Each insert generates 3 events. So, in total 15 events gets generated by this loop.
+    for (int i = 0; i < 5; i++) {
+      run("INSERT INTO TABLE " + dbName + ".unptned values('" + Integer.toString(i) + "')");
+    }
+
+    verifySetup("SELECT count(a) from " + dbName + ".unptned", new String[] {"5"});
```

Comment:
verify of basic sql commands not required, adding this only one place and not others please review the whole test when removing these.